decidim-assemblies/spec/permissions/decidim/assemblies/permissions_spec.rb

```diff
@@ -371,7 +371,7 @@
           { scope: :admin, action: :create, subject: :assembly }
         end
```

Comment:
I think we should add the other examples on what an assembly admin should be able to do with these changes. I'm thinking about the different actions that they should have and the different casuistic (what happens when an assembly admin tries to do actions to a grandmother assembly and things like that). Starting from line 488 could be a starter blueprint in what I had in mind, but we need to add more explicit checks in actions and also that's about "`assemblies types`" example group. What do you think?